.gitignore

```diff
@@ -12,3 +12,4 @@ vendor/
 Guardfile
 pkg
 .DS_Store
+.idea/
```

Comment:
please remove this, this is system specific and should be in your global .gitignore and the change is not related to the PR

Reply:
removed